docs/app/showcase/list_view/dist/app/views/listViews/listView1.html

```diff
@@ -3,70 +3,60 @@
         <multiple-select-actions on-select="vm.select(item);" on-deselect="vm.deselect(item)"
                                  key-fn="vm.keyFn(item)" total="vm.totalItems">
             <div class="row">
-                <div class="col-md-10">
-                    <div class="row">
```

Comment:
These are the changes which were made to the toolbar on the old site, they were missed when moving the showcase pages to open source.